### PR TITLE
[ADD] POS: page to configure and use product combos 

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
Task-3604850

### CODE REVIEW

File: content/applications/sales/point_of_sale/configuration/combos.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Step two is to create a specific product to gather combo choices. To do this:
+
+#. Go to :menuselection:`Point of Sale --> Products --> Products` and click :guilabel:`New`.
+#. Set the :guilabel:`Product Type` to :guilabel:`Combo` and fill in the general information tab.
+
+   .. note::
+      - The sales price you enter will be the price of the combo, regardless of the original
+        prices of the products within the combo choices.
+      - The combo product price is only affected by the extra price optionally defined at the combo
```
Comment (masp-odoo) [inline content/applications/sales/point_of_sale/configuration/combos.rst:33]: Retour à la ligne après combo

File: content/applications/sales/point_of_sale/configuration/combos.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        choice creation or if a variant of one of the items has a specified extra price.
+#. Go to the :guilabel:`Combo Choices` tab, click :guilabel:`Add a line`, and select the
+   combinations to add. You can also create a new combination at this step by clicking
+   :guilabel:`New` on the popup window.
+
+.. image:: combos/combo-product-form.png
+   :scale: 75%
+
+Once you have created and added the combo choices into a product, you can sell combos in your retail
```
Comment (masp-odoo) [inline content/applications/sales/point_of_sale/configuration/combos.rst:42]: Retour à la ligne après retail

File: content/applications/sales/point_of_sale/configuration/combos.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+store or restaurant.
+
+Practical application
+=====================
+
+:ref:`Open a POS session <pos/session-start>` and select the combo product. Choose the options and
```
Comment (masp-odoo) [inline content/applications/sales/point_of_sale/configuration/combos.rst:48]: Retour à la ligne après "and"

**Review by masp-odoo — CHANGES_REQUESTED** [review]
Hello Loredana, très contente de faire cette première review sur POS. Tout est clair, je n'ai rien à redire si ce n'est que les retours à la ligne apparaissent bizarrement chez moi donc je l'ai noté à chaque endroit. À dispo si besoin !

**Review by masp-odoo — APPROVED** [review]

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hello my dear @LoredanaLrpz it was an honor for me to review this PR. 

Haha kidding aside, I added a few suggestions/comments; some can be discarded if you want, some (one) are a little forgetfullnessiness on your part and one is actually something I'd like to discuss with you if that's ok! Let me know when you're ready (tomorrow - Tuesday maybe?).

File: content/applications/sales/point_of_sale/configuration/combos.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+#. Go to :menuselection:`Point of Sale --> Products --> Product Combos` and click :guilabel:`New`.
+#. Name your combo and add the products you want customers to choose from by clicking :guilabel:`Add
+   a line`. You can also include an extra price for each option in the :guilabel:`Extra Price`
+   column.
+
+As a reference, the selected product's original price is displayed in the :guilabel:`Original Price`
+column.
+
+.. image:: combos/combo-form.png
+   :scale: 75%
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/sales/point_of_sale/configuration/combos.rst:23]: 😭

File: content/applications/sales/point_of_sale/configuration/combos.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
+Product combos
+==============
+
+The "Product Combos" feature enables users to define and manage combination options within a
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/configuration/combos.rst:4]: I've changed it, tell me what you think =D

File: content/applications/sales/point_of_sale/configuration/combos.rst [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+#. Go to :menuselection:`Point of Sale --> Products --> Products` and click :guilabel:`New`.
+#. Set the :guilabel:`Product Type` to :guilabel:`Combo` and fill in the  :guilabel:`General
+   Information` tab.
+
+   .. note::
+      - The sales price for the combo product is fixed and does not vary based on the individual
+        prices of included items or the quantity of items in the combo.
+      - The combo product price is only affected by the extra price optionally defined at the combo
+        choice creation or if a variant of one of the items has a specified extra price.
```
Comment (auva-odoo) [inline content/applications/sales/point_of_sale/configuration/combos.rst:42]: Good job rephrasing this one :) I just think we can combine the two bullet points now (+ I'd replace "for" with "of"):
```suggestion
      The sales price of the combo product is fixed and does not vary based on the individual
       prices of included items or the quantity of items in the combo. It is only affected by the extra price optionally 
       defined at the combo choice creation or if a variant of one of the items has a specified extra price.
```

Comment (auva-odoo) [inline content/applications/sales/point_of_sale/configuration/combos.rst:42]: I think we can combine the two bullet points (+ I'd replace "for" with "of"):
```suggestion
      The sales price of the combo product is fixed and does not vary based on the individual
       prices of included items or the quantity of items in the combo. It is only affected by the extra price optionally 
       defined at the combo choice creation or if a variant of one of the items has a specified extra price.
```

**Review by auva-odoo — APPROVED** [review]
Nice job on the intro (and the rest of the PR as well of course), I think that clarifies it :) I just have one more tiny comment but since it's so small I'll go ahead and approve this already :) You can r+ whenever you're ready!
@robodoo delegate+

**Comment by LoredanaLrpz** [comment]
@robodoo r+

**Comment by LoredanaLrpz** [comment]
@robodoo r-

**Comment by LoredanaLrpz** [comment]
@robodoo r+

**Pull request closed without merging**